modules/grpc/common/grpc-dest.cpp

```diff
@@ -288,6 +289,13 @@ grpc_dd_set_protobuf_schema(LogDriver *d, const gchar *proto_path, GList *values
   schema->set_protobuf_schema(proto_path, values);
 }
 
+void
+grpc_dd_set_proto_var(LogDriver *d, LogTemplate *proto_var)
+{
+  GrpcDestDriver *self = (GrpcDestDriver *) d;
```

Comment:
It's out of scope of this PR, but the definition of `struct GrpcDestDriver_` and all other C structs should be in an extern C block.